package.json

```diff
@@ -42,15 +42,18 @@
     "babel-preset-es2015": "^6.24.1",
     "chai": "^3.5.0",
     "eslint": "^3.4.0",
+    "husky": "^0.13.3",
```

Comment:
Hmmmmm.... what do we use it for, then? I don't see any file in `src` or `test` use anything called `husky`, so if our code doesn't use it, and none of the dependencies introduced here use it... why is it a dependency?